docs/pages/repo/docs/core-concepts/monorepos/task-graph.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: The Task Graph
+description: Turborepo builds a Task Graph based on your turbo config
+---
+
+One of the reasons Turborepo is so fast is that it builds a task graph based on
+a combination of your `turbo.json` and package dependencies. Depending on the
+configuration, Turborepo can figure out the order in which tasks need to run,
+and which tasks can run at the same time. This data structure is called a graph,
+or more specifically a [directed acyclic graph][1], or a DAG.
+
+For example, if your `turbo.json` configures a `build` task:
+
+```json
+{
+  "pipeline": {
+    "build": {
+      "dependsOn": ["^build"]
+    }
+  }
+}
+```
+
+and you have an app that depends on 2 packages, `turbo run build` will build a
+task graph that looks like this:
+
+\<insert graphic\>
```

Comment:
@anthonyshew how are your Figma skills? Can you help me make something here? Or maybe we screenshot [your tool](https://www.npmjs.com/package/@ashew/turboviz)?